docs/dev/new_check_howto.md

```diff
@@ -302,7 +302,7 @@ Arguments:
   * *double*
   * *float*
   * *dictionary*
-  * *list\**
+  * *list✱*
```

Comment:
I'm not sure if this was intended, but here is how it looks on local build - https://cl.ly/0b07bcbf8cb2

Reply:
Hmm, you're right, that's not great. Would an HTML code work?
```suggestion
  * *list&#42;*
```

For example?

Reply:
Looks good - https://cl.ly/fa014999c7fb